grammars/prql-lezer/src/prql.grammar

```diff
@@ -30,6 +30,8 @@ TupleExpression { "{" newline* tupleItem (("," newline*) tupleItem)* ","? newlin
 
 tupleItem { DeclarationTuple | expression | CallExpression | CaseBranch }
 
+Annotation { "@{" commaSep<Declaration>? "}" }
```

Comment:
Does it account for the newline? If we have a function below it, does it handle that OK?

Reply:
Oops, no it didn't. Well spotted! Now it does.